server/container/guice/mailbox/src/main/java/org/apache/james/modules/mailbox/DefaultEventModule.java

```diff
@@ -57,6 +57,10 @@ protected void configure() {
         bind(EventDelivery.class).to(InVmEventDelivery.class);
         bind(EventBus.class).to(InVMEventBus.class);
 
+        Multibinder.newSetBinder(binder(), EventBus.class)
+            .addBinding()
+            .to(EventBus.class);
```

Comment:
.to(InVMEventBus.class) is better?

Reply:
No I prefer define the targeted binding here rather than the source one.